### DISCUSSION TITLE
Solaris 11.4 x86 running on Mac M1 pro using UTM

### DISCUSSION THREAD

**Discussion by wonkyworld** [utmapp/UTM#5752] [UTM Help (macOS)]
Has anyone tried and got successful in deploying Solaris 11.4 x86 on Mac M1 pro (Ventura 13.5) using UTM version 4.3.5 (87)?

I'm using emulate and have tried using the System settings below,

Standard PC (pc-q35-7.2) -
Unchecked UEFI Boot
Left RNG Device as it is (checked)
USB Support - Tried usb2 and usb3

1st attempt, failed  with errors below

WARNING: /pci@0,0/pci1af4,1100@1d,1 (uhci1): No SOF interrupts have been received, this ISB UHCI host controller is unusable
SATA disk device at port 0 does not support LBA

Then goes into maintenance mode
Enter user name for system maintenance (control-d to bypass):

2nd Attempt below:
Standard PC (pc-q35-7.2) -
Unchecked UEFI Boot
Left RNG Device as it is (checked)
USB Support - Disabled

2nd attempt failed with errors below:
SATA disk device at port 0 does not support LBA

Then goes into maintenance mode
<img width="962" alt="image" src="https://github.com/utmapp/UTM/assets/146420514/ea4ede94-aee1-45af-9579-48783e60385e">

3rd Attempt below:
Standard PC (i440FX + PIIX, 1996) 7.2
Unchecked UEFI Boot
Left RNG Device as it is (checked)
USB Support - Tried usb2 and usb3

Third attempt allowed me to boot to solaris 11 x86 iso (text-installer) got to the installation process. But only up to 99% and stuck after that.

**Comment by salvomic** [comment]
Thanks to your observations today I tried in UTM 4.4.4(92), changing some parameters. It finally got installed on my new macBook pro M3 pro and now I installed Gnome (sudo pkg install gnome-desktop) and configured. 

Architecture: x86_64
UEFI boot unchecked
RNG Device checked
Ballon Device checked
Use local time for base clock checked
Machine (computer): Standard PC (i440FX + PIIX, 1996) (default) (pc-i440fx-7.2) 
CPU: AMD Opteron 240 (Gen 1 Class Opteron) (Opteron_G1-v1)
Select an option to exclude: mca
CPU core: default 
4 GB RAM
USB 2
Display Video: virtio-vga
It was important to choose the network adapter: Intel Gigabit Ethernet (e1000) shared
Sound: Intel 82801AA AC97 Audio (AC97)

At the boot it presents an error complaining about BIOS microcode patch for AMD Athlon 64/Opteron the absence of something about AMD, but then it continues after a pause.

the boot is so sloooow :-(

**Comment by larseggert** [comment]
> Third attempt allowed me to boot to solaris 11 x86 iso (text-installer) got to the installation process. But only up to 99% and stuck after that.

It'll took an hour or two sitting at 99% for it to complete, but it did complete eventually. You'll see that the qcow2 image keeps growing while it's sitting at 99%.

**Comment by zyv** [comment]
I've had success with:

* Standard PC (i440FX + PIIX, 1996)
* 8192 MiB RAM
* 4 Cores
* UEFI Boot disabled
* Display: virtio-vga
* Network: virtio-net-pci
* Disk: IDE

Everything seems to be working. As soon as I switch disk to VirtIO, it gets stuck in a boot loop.

**Reply by zyv** [reply]
OpenIndiana is kind of the same, but

* Disable USB
* Disk: VirtIO

Multicore is working on the latest version!
